docs/topics/proc_getting-the-secured-booster-api-endpoint.adoc

```diff
@@ -50,3 +50,4 @@ A sample output is shown in the following table:
 ====
 --
 
+IMPORTANT: `PROJECT_ID` is based on the name you entered when generating your booster using link:{link-launcher-oso}[{launcher-oso}] or the {launcher} tool. The default value for `PROJECT_ID` is `{parameter-runtime}-rest-http-secured`.
```

Comment:
The concept of "default value" IMHO doesn't make a lot of sense to me here, and the value isn't actually correct :-) See https://github.com/openshiftio-vertx-boosters/vertx-secured-http-booster/blob/master/pom.xml#L29